setup.py

```diff
@@ -0,0 +1,24 @@
+import setuptools
+
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setup_info = {
+    "name": "grepper-python",
+    "version": "0.0.1a",
```

Comment:
With `pyproject.toml` you'll be able to set the version dynamically, which will remove this duplicate version string and avoid errors :)

Reply:
that sounds super handy! looks like im gonna have to learn it after all